packages/compass-components/src/components/breadcrumb.tsx

```diff
@@ -0,0 +1,85 @@
+import React from 'react';
+import { spacing } from '@leafygreen-ui/tokens';
+import { css, cx } from '@leafygreen-ui/emotion';
+import { palette } from '@leafygreen-ui/palette';
+import { useDarkMode } from '../hooks/use-theme';
+import { Link, Icon, Body } from './leafygreen';
+
+export type BreadcrumbItem = {
+  name: string;
+  onClick: () => void;
+};
+
+const breadcrumbStyles = css({
+  display: 'flex',
+  gap: spacing[1],
+  alignItems: 'center',
+  height: spacing[4],
+});
+
+const dbLinkLightStyles = css({
+  color: palette.green.dark2,
+});
+
+const dbLinkDarkStyles = css({
+  color: palette.green.base,
+});
+
+const textStyles = css({
+  overflow: 'hidden',
+  textOverflow: 'ellipsis',
+  whiteSpace: 'nowrap',
+  fontWeight: 'bold',
+});
+
+const lastItemStylesLight = css({
+  color: palette.gray.dark1,
+});
+
+const lastItemStylesDark = css({
+  color: palette.gray.base,
+});
+
+export const Breadcrumbs = ({ items }: { items: Array<BreadcrumbItem> }) => {
+  const darkMode = useDarkMode();
+  return (
+    <div className={breadcrumbStyles} data-testid="breadcrumbs">
```

Comment:
We probably want to have some sort of overflow control here:

![image](https://github.com/mongodb-js/compass/assets/5036933/d4f55eff-5f38-45a4-9bc5-21a756eef41c)

Reply:
Resolved in [7f402b9](https://github.com/mongodb-js/compass/pull/5501/commits/7f402b9416ae32c2d99c6a0b2295ad7009838dee)